signup/static/js/djaodjin-signup-vue.js

```diff
@@ -120,8 +123,52 @@ var app = new Vue({
                 vm.apiKey = "ERROR";
                 showErrorMessages(resp);
             });
+        },
+        getContact: function(cb){
+            var vm = this;
+            $.ajax({
+                method: 'GET',
+                url: djaodjinSettings.urls.user.api_contact,
+            }).done(function(res) {
+                console.log(res)
```

Comment:
`console.log` statement shouldn't be committed. They have a tendency to break code when run on IE.